pantheon/src/main/java/tech/pegasys/pantheon/controller/IbftLegacyPantheonController.java

```diff
@@ -176,6 +176,11 @@ private IbftLegacyPantheonController(
           } catch (final IOException e) {
             LOG.error("Failed to close storage provider", e);
           }
+          try {
```

Comment:
We should stop the synchronizer prior to the storage.  The shutdown process could try to write more data.